external/anomaly/tests/config.py

```diff
@@ -0,0 +1,7 @@
+try:
+    import os
+    from e2e import config as config_e2e
+
+    config_e2e.repository_name = os.environ.get("TT_REPOSITORY_NAME", "training_extensions/external/anomaly")
+except ImportError:
+    pass
```

Comment:
Please, remove this file at all.

You already does this operation in `external/anomaly/tests/conftest.py` -- you calls the function `ote_conftest_insertion`

Reply:
removed